scripts/verify_sitemap_links.py

```
@@ -0,0 +1,123 @@
+        print(f"[+] GitHub Pages URL OK (Source exists on disk, pre-merge): {url} -> {disk_file}")
+        return True
+
+    print(f"[-] GitHub Pages URL does not exist on live site AND has no disk source: {url}")
+    return False
+
+def main() -> None:
+    print("[*] Starting Sitemap and Link Integrity Verification...")
+
+    # 1. Verify sitemap.txt URLs
+    try:
+        with open("sitemap.txt", "r") as f:
```

Ruff (0.16.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)